Controller/AdminController.php

```diff
@@ -313,10 +313,16 @@ public function modifyForumAction(Request $request, $id = null)
             $forum = new ForumEntity();   
         }
         
+        $topiccount = ModUtil::apiFunc('ZikulaDizkusModule', 'user', 'countstats', ['id' => $id,
```

Comment:
stop using `ModUtil::apiFunc` please. you or someone will just have to update it again later. It will not be available in 2.0. Write a helper class that can be a service or use a repository class.